docs/reference/commandline/network_inspect.md

```diff
@@ -201,6 +201,97 @@ $ docker network inspect ingress
 ]
 ```
 
+### Using `verbose` option for `network inspect`
+
+`docker network inspect --verbose` for swarm mode overlay networks shows service-specific details like VIP and port mappings. It also
```

Comment:
If I didn't have the context of this conversation, I would read that `its` as meaning that the *network*'s VIP and port mappings are shown.